### DISCUSSION TITLE
Curation report Feb 13-17

### DISCUSSION THREAD

**Discussion by susancoort** [wikipathways/GPML-Repo#115] [Curation reports]
A new cholesterol biosynthesis pathway was created: [WP5329](https://classic.wikipathways.org/index.php/Pathway:WP5329). This pathway has colored nodes, should we advise the author to remove the colors? 
WikiPathways has several pathways that contain cholesterol biosynthesis. Do we want to keep them all, or should we merge because of the overlap? (Part of) the cholesterol biosynthesis pathways are:
-	Cholesterol metabolism [WP5304](https://classic.wikipathways.org/index.php/Pathway:WP5304)
-	Cholesterol metabolism with Bloch and Kandutsch-Russell pathways [WP4718](https://classic.wikipathways.org/index.php/Pathway:WP4718)
-	Mevalonate arm of cholesterol biosynthesis pathway [WP4190](https://classic.wikipathways.org/index.php/Pathway:WP4190)
-	Cholesterol biosynthesis, regulation, and transport [WP4281](https://classic.wikipathways.org/index.php/Pathway:WP4281)

Added ontology tags and updated tags.

The newly created pathway on fatty acids and lipoprotein: [WP5323](https://classic.wikipathways.org/index.php/Pathway:WP5323) has a lot of overlap with the recently created Cholesterol metabolism pathways [WP5304](https://classic.wikipathways.org/index.php/Pathway:WP5304). Do we want to have both pathways? 

Do we still want to test the search on the classic WikiPathways website?  I would change this to test the **browse** on the new website for newly **approved** pathways. 

With kind regards,
Susan

@andrawaag you are next!

**Comment by khanspers** [comment]
Thanks @susancoort! I have contacted the author of WP5329 and WP5323 about the issues with having color-filled nodes. 

In terms of pathway overlap, I think we should only suggest removing pathways that are clearly a 1-to-1 copy of content. Also, WP5323 seems to be specific to hepatocytes (same for WP5329). 

For the protocol, yes maybe it doesn't make sense to be testing if the search works on the classic site. Or does whether a pathway is indexed on classic.wikipathways.org affect web service results? If yes for the latter, then we still need to check for it. @mkutmon @AlexanderPico 

I will update the "qaprotocol-extras" task to check if the [newly approved pathways](https://classic.wikipathways.org/index.php?title=Special:CurationTags&showPathwaysFor=Curation:AnalysisCollection) are present in the [table](https://www.wikipathways.org/browse/table.html) at the new site (table is easier to check than browse). Does that make sense?

**Reply by khanspers** [reply]
Actually, I don't know if it's really necessary for curators to be checking newly approved pathways on the new site. Wouldn't we already know if there's a problem based on if the [daily sync action](https://github.com/wikipathways/wikipathways-database/actions/workflows/scheduled_gpml_sync.yml) fails?

**Reply by susancoort** [reply]
Dear @khanspers, thank you for your response.

I was a bit worried that we have too many cholesterol-related pathways. Within the cholesterol metabolism pathway  (WP5304) the hepatocyte part is also present, so there is an overlap with WP5329. 

Will the daily sync action tell you whether you can find a newly approved pathway? It will only tell you whether the syncing worked, right?

**Reply by Chris-Evelo** [reply]
Just a note. Please keep in mind that the same content can sometimes be relevant for more than one community and they feel responsible for their "own" pathway, tagged as belonging to that community. One of these pathways is for instance from Lipid Maps. That typically results in different focus and different styles too. It will typically be hard to merge and keep the community involvement.

**Reply by susancoort** [reply]
Just as a small remark. The cholesterol metabolism pathway was created by my student Tom and we assigned it to the lipid maps community since it was mentioned that this pathway is also relevant to the community.

**Comment by khanspers** [comment]
Regarding the question of what curators should be checking for in terms of newly approved pathways / indexed pathways, to make it clear there are two separate issues:
1. Is it still necessary to check if indexing is working on classic.wikipathways.org? Does this affect web services? If so, then we should keep checking. If not, we can remove this task from the QA protocol
2. Is it necessary for curators to check if newly [approved pathways on classic](https://classic.wikipathways.org/index.php?title=Special:CurationTags&showPathwaysFor=Curation:AnalysisCollection) (sort by latest tag date) are present in [New pathways list](https://www.wikipathways.org/browse/new.html)? Or can we assume that newly approved pathways are there if the daily sync worked? Maybe the answer is that it would make sense to have this extra check by curators during this initial transition phase.

**Reply by khanspers** [reply]
We decided **yes** on 1 and **no** on 2, we will keep checking if search works on classic while we have it running, and curators don't need to check newly added on the new site.
